src/scope/Operator.js

```diff
@@ -16,6 +16,10 @@ import User from '../entity/User'
 import Batch from '../entity/Batch'
 import Place from '../entity/Place'
 import File from '../entity/File'
+import AccessPolicy from '../entity/AccessPolicy'
+import AccessToken from '../entity/AccessToken'
+import Me from '../entity/Me'
+import Access from '../entity/Access'
```

Comment:
I missed this, `Access` not required directly on `Operator` scope - sub-resource of `Account`

Reply:
fixed, thanks